mix.exs

```diff
@@ -30,7 +30,7 @@ defmodule AntikytheraAws.MixProject do
       {:antikythera, "~> 0.2"},
       {:ex_doc     , "~> 0.18", [only: :dev , runtime: false]},
       {:dialyxir   , "~> 0.5" , [only: :dev , runtime: false]},
-      {:credo      , "~> 0.8" , [only: :dev , runtime: false]},
+      {:credo      , "~> 1.2.0" , [only: :dev , runtime: false]},
```

Comment:
Antikythera is using 1.4.0, so it would be better to use `~> 1.4.0` although it gets 1.4.1 now.

Reply:
Yes, it is better to follow the credo version a9a uses
Fixed in https://github.com/access-company/antikythera_aws/pull/8/commits/bd5f3c36454a35f826e1cea30585d241664ae9ca

- By upgrading to v1.4.1, the following warning occurred [here](https://github.com/access-company/antikythera_aws/commit/bd5f3c36454a35f826e1cea30585d241664ae9ca#diff-1d0e27fa5fd3325a91ce05f8f00ea69fee09a4df2e3e9eb56fb1ee833dc51f04L106).
```
  Warnings - please take a look
┃
┃ [W] ↗ Prefer System.cmd/2,3 over :os.cmd/1 to prevent command injection
┃       lib/s3.ex:107 #(AntikytheraAws.S3.AssetStorage)

```

- I fixed it and confirmed that `Mix.Tasks.AntikytheraCore.UploadNewAssetVersions` succeeds as usual in the internal environment.
- This is a build log of dodai_console by Jenkins and we can confirm that `dodai_console/css/style_024a8d46b911709f30c817a4be29a920.css` and  `dodai_console/js/app_98e7a9207ba696366bce2ba261d27280.js` are uploaded in the compressed state because their `gzip?` flag is `true`.
```
+(2021-02-25 07:36:01+00:00) mix antikythera_core.upload_new_asset_versions
uploaded priv/static/css/style.css as key=dodai_console/css/style_024a8d46b911709f30c817a4be29a920.css mime=text
/css gzip?=true
uploaded priv/static/img/ac_logo.png as key=dodai_console/img/ac_logo_80117f76e25726c19821a81d8bbfdb54.png mime=
image/png gzip?=false
uploaded priv/static/img/ac_logo_L.png as key=dodai_console/img/ac_logo_L_ebc73ee291751712127f302a1bcccbdc.png m
ime=image/png gzip?=false
uploaded priv/static/img/account_logout.png as key=dodai_console/img/account_logout_04500bbc2eca897277fb36454fe4
b486.png mime=image/png gzip?=false
uploaded priv/static/img/account_setting.png as key=dodai_console/img/account_setting_1f84eb62f7c85588064b44bdfc79a718.png mime=image/png gzip?=false
uploaded priv/static/img/arrow_account.png as key=dodai_console/img/arrow_account_a0199c5fa7b5cc70755c030328df89b7.png mime=image/png gzip?=false
uploaded priv/static/img/arrow_account_hover.png as key=dodai_console/img/arrow_account_hover_3cd0ff1b5bb9181c4b477d05d8dc9682.png mime=image/png gzip?=false
uploaded priv/static/img/icon_account.png as key=dodai_console/img/icon_account_cd151dc90d584adca1856a9f9c93e210.png mime=image/png gzip?=false
uploaded priv/static/img/icon_app.png as key=dodai_console/img/icon_app_1f364beb5e31d6073d7247e5f5851ec7.png mime=image/png gzip?=false
uploaded priv/static/img/icon_del.png as key=dodai_console/img/icon_del_77e14fe8433fda848234cebfaeb216fa.png mime=image/png gzip?=false
uploaded priv/static/img/icon_header_back.png as key=dodai_console/img/icon_header_back_61560c07aa8ad106bb44a67c69504842.png mime=image/png gzip?=false
uploaded priv/static/img/icon_header_menu.png as key=dodai_console/img/icon_header_menu_a38b148caa66ace561d57597f063d520.png mime=image/png gzip?=false
uploaded priv/static/img/icon_search.png as key=dodai_console/img/icon_search_80d9eb4ab382e7feaf8d1caa624e8f90.png mime=image/png gzip?=false
uploaded priv/static/img/icon_star.png as key=dodai_console/img/icon_star_aa4ee5e9ccc8ab7a35e7a83eb3fac026.png mime=image/png gzip?=false
uploaded priv/static/img/icon_star_off.png as key=dodai_console/img/icon_star_off_bebc0f4b990fb8c9fa1e0db3c981f518.png mime=image/png gzip?=false
uploaded priv/static/js/app.js as key=dodai_console/js/app_98e7a9207ba696366bce2ba261d27280.js mime=application/javascript gzip?=true
created an asset list file: /home/solomondeploy/antikythera_gear_asset_list/dodai_console-0.0.1-20210225065557+155090697cd7dfdf675fac75d533d06f0e0f8d22
Updated 3 paths from the index
```
- In AWS Management Console, I confirmed that these files are uploaded successfully to S3.
- I confirmed that we can use dodai_console as usual.